src/index.ts

```diff
@@ -11,6 +11,7 @@ export { transformDimension } from './transformDimension.js';
 export { transformFontWeight } from './transformFontWeight.js';
 export { transformColorModifiers } from './color-modifiers/transformColorModifiers.js';
 export { transformLineHeight } from './transformLineHeight.js';
+export { TransformOptions } from './TransformOptions.js';
```

Comment:
```suggestion
export type { TransformOptions } from './TransformOptions.js';
```

Since it's a type-only export, specifying that helps the typescript compiler not make mistakes when it's ambiguous whether something is a type or a value